satellite/orders/service.go

```diff
@@ -611,7 +621,9 @@ func (service *Service) CreatePutRepairOrderLimits(ctx context.Context, repairer
 		return nil, Error.Wrap(err)
 	}
 	pieceSize := eestream.CalcPieceSize(pointer.GetSegmentSize(), redundancy)
-	totalPieces := redundancy.TotalCount()
+	totalPiecesToRepair := int(math.Ceil(
+		float64(redundancy.OptimalThreshold()) * (1 + optimalThresholdExceessRate),
```

Comment:
Make sure that `totalPiecesToRepair` isn't over `totalPieces` otherwise set it to `totalPieces`